plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -108,3 +108,8 @@ Luma.LumaMax="Luma Max"
 Luma.LumaMin="Luma Min"
 Luma.LumaMaxSmooth="Luma Max Smooth"
 Luma.LumaMinSmooth="Luma Min Smooth"
+NvidiaGreenscreenFilter="NVIDIA RTX Background Removal"
```

Comment:
Apparently the actual feature name is "NVIDIA Virtual Background". If you change this, just changing this string should be fine, as this one is user-facing.

Reply:
it's more complicated. In the sdk, the FX used is called: AI Greenscreen (https://docs.nvidia.com/deeplearning/maxine/vfx-sdk-programming-guide/index.html).
The FX just creates an alpha image and nothing more. This alpha can then be used to generate:
- a virtual greenscreen,
- to remove the background,
- to swap the background for another (= virtual background).
The implementation is left to the sdk user.
I've selected to remove the background. There is no function in the sdk which provides that FX, I had to code it w/ obs own rendering functions through an hlsl effect.
 In the RTX Broadcast app, Nvidia elected to call that: Background Removal.
This is the best descriptive imo since it's not a virtual background at all.

Reply:
here's feedback from Gerardo : `for the public name we would prefer NVIDIA Background Removal, or NVIDIA Virtual Background (either works)`
I'll modify the string to NVIDIA Background Removal then.

Reply:
Sounds good to me. Thanks!